scripts/work_jobs.py

```
@@ -223,25 +242,70 @@ def handle_pretrade_check(job: Job, engine: Engine) -> dict:
+        raise RuntimeError(
+            "BROKER_TWS_PORT is not set and no port was provided in job payload."
+        )
+
+    if isinstance(client_id_raw, int):
+        client_id = client_id_raw
+    else:
+        client_id = get_int_env("BROKER_TWS_QUOTES_CLIENT_ID", 141)
+    if client_id is None:
+        raise RuntimeError(
+            "BROKER_TWS_QUOTES_CLIENT_ID is not set and no client_id was provided in job payload."
+        )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)